src/App.tsx

```diff
@@ -1,25 +1,106 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | undefined {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || undefined;
+}
```

Comment:
Would be better to use null here 
```suggestion
function getUser(userId: number): User | null {
  const foundUser = usersFromServer.find(user => user.id === userId);

  return foundUser || null;
}
```